### PR TITLE
Update sklearn notebooks to use 0.23-1

### PR DESCRIPTION
**Pull request by edwardjkim**
*Issue #, if available:*

*Description of changes:*

The latest scikit-learn container `0.23-1` has recently been released. This PR updates three scikit-learn notebooks to use the latest image.

`Inference Pipeline with Scikit-learn and Linear Learner.ipynb` required changes in the script because `ColumnTransformer` began checking the number of features in scikit-learn 0.22. The previous script had an extra column (the label column) in the featurizer, and `ColumnTransformer` ignore the extra column in 0.21, but it raises an error in 0.23.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by edwardjkim** [comment]
@iyerr3 Can you please review and merge?
